doc/redirects.json

```diff
@@ -2710,17 +2710,17 @@
   "waf-hook-honored-variables": [
     "index.html#waf-hook-honored-variables"
   ],
-  "zig-hook": [
-    "index.html#zig-hook"
+  "zig": [
+    "index.html#zig"
   ],
-  "zig-hook-example-code-snippet": [
-    "index.html#zig-hook-example-code-snippet"
+  "zig-example-code-snippet": [
+    "index.html#zig-example-code-snippet"
   ],
-  "zig-hook-variables-controlling": [
-    "index.html#zig-hook-variables-controlling"
+  "zig-variables-controlling": [
+    "index.html#zig-variables-controlling"
   ],
-  "zig-hook-exclusive-variables": [
-    "index.html#zig-hook-exclusive-variables"
+  "zig-exclusive-variables": [
+    "index.html#zig-exclusive-variables"
```

Comment:
You should retain the redirects from the old anchors to the new ones. I've typed out what this file should look like for you:
```json
  "zig": [
    "index.html#zig",
    "index.html#zig-hook"
  ],
  "zig-example-code-snippet": [
    "index.html#zig-example-code-snippet",
    "index.html#zig-hook-example-code-snippet"
  ],
  "zig-variables-controlling": [
    "index.html#zig-variables-controlling",
    "index.html#zig-hook-variables-controlling"
  ],
  "zig-exclusive-variables": [
    "index.html#zig-exclusive-variables",
    "index.html#zig-hook-exclusive-variables"
  ],
```

Reply:
thanks! this should be addressed